cargo-powdr/template/guest/src/main.rs

```diff
@@ -0,0 +1,12 @@
+use powdr_riscv_runtime;
+use powdr_riscv_runtime::io::read;
+
+fn main() {
+    // Read some data from channel 1.
```

Comment:
this could be more specific maybe? could be nice to make it clear that this is a `serde` object

Reply:
added a comment above this one